src/geocat/viz/util/util.py

```diff
@@ -119,71 +149,84 @@ def add_major_minor_ticks(ax,
+        Text font size for lefttitle. A default value of 18 is used if nothing is set.
 
-        maintitlefontsize (:class:`int`):
-            Text font size for maintitle. A default value of 18 is used if nothing is set.
+    righttitle: :class:`str`
+        Text to use for an optional right-aligned title, if any. For most plots, only a maintitle is enough,
+        but for some plot types, a righttitle likely with a left-aligned title, lefttitle, can be used together.
 
-        lefttitle (:class:`str`):
-            Text to use for an optional left-aligned title, if any. For most plots, only a maintitle is enough,
-            but for some plot types, a lefttitle likely with a right-aligned title, righttitle, can be used together.
+    righttitlefontsize: :class:`int`
+        Text font size for righttitle. A default value of 18 is used if nothing is set.
 
-        lefttitlefontsize (:class:`int`):
-            Text font size for lefttitle. A default value of 18 is used if nothing is set.
+    xlabel: :class:`str`
+        Text for the x-axis label.
 
-        righttitle (:class:`str`):
-            Text to use for an optional right-aligned title, if any. For most plots, only a maintitle is enough,
-            but for some plot types, a righttitle likely with a left-aligned title, lefttitle, can be used together.
+    ylabel: :class:`str`
+        Text for the y-axis label.
 
-        righttitlefontsize (:class:`int`):
-            Text font size for righttitle. A default value of 18 is used if nothing is set.
+    labelfontsize: :class:`int`
+        Text font size for x- and y-axes. A default value of 16 is used if nothing is set.
+        
+    Notes
+    -----
+    If no lefttitle and righttitle is set, maintitle is placed just top to the axes as follows:
 
-        xlabel (:class:`str`):
-            Text for the x-axis label.
+    >>>                  maintitle
+    >>>  ___________________________________________
+    >>> |                   Axes                    |
+    >>> |                                           |
 
-        ylabel (:class:`str`):
-            Text for the y-axis label.
+    
+    If any of lefttitle or righttitle is set, lefttitle and righttitle are placed into a row that is just top to the axes, and maintitle is placed to top of
+    the row of lefttitle/righttitle as follows:
 
-        labelfontsize (:class:`int`):
-            Text font size for x- and y-axes. A default value of 16 is used if nothing is set.
+    >>>                  maintitle
+    >>>  lefttitle                        righttitle
+    >>>  ___________________________________________
+    >>> |                   Axes                    |
+    >>> |                                           |
+        
+    Examples
+    --------
+    All usage examples are within the GeoCAT-Examples Gallery. To see more usage cases, search the function on the `website <https://geocat-examples.readthedocs.io/en/latest/index.html>`_.
```

Comment:
```suggestion
    All usage examples are within the GeoCAT-Examples Gallery. To see more usage cases, search the function on the 
    `website <https://geocat-examples.readthedocs.io/en/latest/index.html>`_.
```
Same as above wrt line length